### PR TITLE
Re-enables pylint consider-using-fstring check

### PR DESCRIPTION
**Pull request by dosaboy**

### CODE REVIEW

File: hotsos/core/plugins/kernel/kernlog/calltrace.py [hunk 1]
```diff
@@ -64,15 +61,15 @@ def __repr__(self):
...
+                    s.append(f">> node{node}")
                     for zone, fields in zones.items():
-                        s.append(" {}: {}".format(zone, repr(fields)))
+                        s.append(f" {zone}: {repr(fields)}")
             else:
-                s.append("{}{}: {}".format(prefix, k, v))
+                s.append(f"{prefix}{k}: {v}")
 
-        return "\n{}\n".format('\n'.join(s))
+        return "\n{}\n".format('\n'.join(s))  # noqa, pylint: disable=consider-using-f-string
```
Comment (mustafakemalgilor) [inline hotsos/core/plugins/kernel/kernlog/calltrace.py:72]: Why we're suppressing the warning here?

```
f"\n{'\n'.join(s)}\n"
```
should behave the same:

```python
>>> s=['this', 'is', 'a', 'test']
>>> "\n{}\n".format('\n'.join(s))
'\nthis\nis\na\ntest\n'
>>> f"\n{'\n'.join(s)}\n"
'\nthis\nis\na\ntest\n'

```

File: hotsos/core/ycheck/common.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def __init__(self, key, all_keys):
         self.all_keys = all_keys
 
     def __str__(self):
-        return (f"'{self.key}' key already exists in search registry. "
+        return (f"'{self.key}' key already exists in search registry. "  # noqa, pylint: disable=consider-using-f-string
```
Comment (mustafakemalgilor) [inline hotsos/core/ycheck/common.py:20]: The same as above.

File: hotsos/plugin_extensions/openstack/agent/exceptions.py [hunk 3]
```diff
@@ -210,17 +210,15 @@ def _load(self):
...
-                prefix_match = (r'(?:{})?'.
-                                format(wsgi_prefix or keystone_prefix))
+                prefix_match = rf'(?:{wsgi_prefix or keystone_prefix})?'
 
             # Sometimes the exception is printed with just the class name
             # and sometimes it is printed with a full import path e.g.
             # MyExc or somemod.MyExc so we need to account for both.
             exc_obj_full_path_match = r'(?:\S+\.)?'
-            expr_template = (r"^{}([0-9\-]+) (\S+) .+\S+\s({}{{}})[\s:\.]".
+            expr_template = (r"^{}([0-9\-]+) (\S+) .+\S+\s({}{{}})[\s:\.]".  # noqa, pylint: disable=consider-using-f-string
```
Comment (mustafakemalgilor) [inline hotsos/plugin_extensions/openstack/agent/exceptions.py:221]: Same as above.

File: hotsos/plugin_extensions/rabbitmq/summary.py [hunk 4]
```diff
@@ -29,11 +29,11 @@ def queue_info(self):
             for node, vhost_dist in vhost.node_queue_distributions.items():
                 if vhost_dist['queues']:
                     vhost_queue_dists[vhost.name][node] = (
-                        "{:d} ({:.2f}%)".format(vhost_dist['queues'],
+                        "{:d} ({:.2f}%)".format(vhost_dist['queues'],   # noqa, pylint: disable=consider-using-f-string
```
Comment (mustafakemalgilor) [inline hotsos/plugin_extensions/rabbitmq/summary.py:32]: The same.

File: hotsos/core/plugins/kernel/kernlog/calltrace.py (hunk 1 above)
Comment (dosaboy) [reply hotsos/core/plugins/kernel/kernlog/calltrace.py:72]: it failed when I tried it but perhaps I did it wrong, ill try again.

Comment (mustafakemalgilor) [reply hotsos/core/plugins/kernel/kernlog/calltrace.py:72]: nevermind, it's an error in py3.12 but supported in >=3.12, hence the confusion.

**Review by mustafakemalgilor — APPROVED** [review]
LGTM

File: hotsos/core/plugins/kernel/kernlog/calltrace.py (hunk 1 above)
Comment (dosaboy) [reply hotsos/core/plugins/kernel/kernlog/calltrace.py:72]: so yeah, not supported < py3.12 https://peps.python.org/pep-0701/

Comment (dosaboy) [reply hotsos/core/plugins/kernel/kernlog/calltrace.py:72]: guess we'll have to stick with disables until we grow up to be >= py312 ;)

Comment (mustafakemalgilor) [reply hotsos/core/plugins/kernel/kernlog/calltrace.py:73]: there's a workaround though:

```python
 s_w_newlines = '\n'.join(s)
 return "\n{s_w_newlines}\n"
```

**Pull request merged**